moveit_core/collision_detection_fcl/test/test_fcl_env.cpp

```diff
@@ -0,0 +1,307 @@
+#include <gtest/gtest.h>
+#include <ros/ros.h>
+
+#include <moveit/collision_detection/collision_common.h>
+
+#include <moveit/robot_model/robot_model.h>
+#include <moveit/robot_state/robot_state.h>
+#include <moveit/utils/robot_model_test_utils.h>
+
+#include <moveit/collision_detection_fcl/collision_common.h>
+#include <moveit/collision_detection_fcl/collision_env_fcl.h>
+
+#include <urdf_parser/urdf_parser.h>
+#include <geometric_shapes/shape_operations.h>
+
+/** \brief Brings the panda robot in user defined home position */
+inline void setToHome(robot_state::RobotState& panda_state)
+{
+  panda_state.setToDefaultValues();
+  double joint2 = -0.785;
+  double joint4 = -2.356;
+  double joint6 = 1.571;
+  double joint7 = 0.785;
+  panda_state.setJointPositions("panda_joint2", &joint2);
+  panda_state.setJointPositions("panda_joint4", &joint4);
+  panda_state.setJointPositions("panda_joint6", &joint6);
+  panda_state.setJointPositions("panda_joint7", &joint7);
+  panda_state.update();
```

Comment:
Doesn't this need to set all other joints to 0 explicitly?

Reply:
No, this happens through setToDefaultValues()